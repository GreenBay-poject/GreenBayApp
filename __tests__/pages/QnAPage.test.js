import React from 'react';
import renderer from 'react-test-renderer';
import QnAPage from '../../pages/qnapage'

describe('<QnAPage />', () => {
    it('Sample QnAPage', () => {
        const tree = renderer.create(<QnAPage />).toJSON();
        expect(tree).toMatchSnapshot();
        //expect(tree.children[0].children[0].type).toBe('View');
        //expect(tree.children[0].children[1].type).toBe('View');
        //console.log(tree)

    });
});