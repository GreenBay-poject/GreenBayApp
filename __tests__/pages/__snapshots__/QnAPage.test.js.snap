// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<QnAPage /> Sample QnAPage 1`] = `
<RNCSafeAreaView>
  <View
    accessibilityRole="button"
    accessibilityState={Object {}}
    accessible={true}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "opacity": 1,
      }
    }
  >
    <View
      style={
        Array [
          Object {},
        ]
      }
    >
      <Text
        style={
          Array [
            Object {
              "color": "#007AFF",
              "fontSize": 18,
              "margin": 8,
              "textAlign": "center",
            },
            Object {
              "color": "#153d37",
            },
          ]
        }
      >
        SIGN IN
      </Text>
    </View>
  </View>
</RNCSafeAreaView>
`;
